lib/serverengine/process_manager.rb

```diff
@@ -204,14 +219,31 @@ def tick(blocking_timeout=0)
         return nil
       end
 
-      ready_pipes, _, _ = IO.select(@rpipes.keys, nil, nil, blocking_timeout)
+      if ServerEngine.windows?
+        @rpipes.each{|r, m|
```

Comment:
if my understanding is correct, pipe won't be invalid even when a connecting process exists because another client can connect there. That's the reason why EOFError doesn't happen.

As I commented on https://github.com/fluent/serverengine/pull/26#discussion_r44332439, another idea is to not implement heartbeat at all on windows. I think it's good enough and make it simple. it mans that `ProcessManager#tick` does nothing excepting `@monitors.delete_if {|m| !m.tick(time) }`. Any thoughts there?